index.html

```diff
@@ -296,6 +296,45 @@ <h4>Skills</h4>
             </div>
         </section>
     </main>
+    <section id="contact-form">
+        <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+        <form action="https://formspree.io/f/xeqnpraa" method="post">
+            <input type="text" name="name" placeholder="Full name" maxlength="30" required>
+            <input type="email" name="email" placeholder="Email address" required>
+            <textarea name="message" maxlength="500" placeholder="Enter text here"></textarea>
+            <button type="submit">Get in touch</button>
```

Comment:
- [ ] Great job with the contact section. everything works as expected. nevertheless, kindly add the bottom margin so as to make it look more like the figma version
your version
![Screenshot (207)](https://user-images.githubusercontent.com/65298048/177429673-5ca3fdae-d524-460d-9078-4775ee27c8f1.png)
figma version 
![Screenshot (208)_LI](https://user-images.githubusercontent.com/65298048/177429762-73dcbeab-f5d5-4ccc-b8a5-92ea4c70a7b4.jpg)